_posts/2016-12-19-devlab-v3-is-here.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "Devlab v3 Is Here"
+date: 2015-12-19 8:00:00
+categories: docker devlab development
+author: kent.safranski
+cover: /assets/images/covers/container-load.jpg
+description: After a year of dogfooding, a complete rebuild, and 110+ commits, we are excited to release Devlab v3.
+comments: true
+---
+
+After a year of dogfooding, a complete rebuild, and 110+ commits, we are excited to release Devlab v3.
+
+## New to Devlab?
+
+Devlab is a CLI tool that allows you to easily spin up [Docker](https://www.docker.com/) containers with all the services, environment variables, and settings you need to run a project in a "cleanroom".
+
+Simply put, if you want to run an application you're developing that requires a database (or any other services), you simply define the services, the configuration, and tasks you routinely run. Once your project is configured you run `tasks` and every time Devlab spins up a parent container and links in any service containers you need.
```

Comment:
Not to bikeshed, but the "Simply put" bothers me here (and in the readme). What follows is actually multiple fairly complex statements that are probably pretty befuddling, especially to folks new to docker.  Combined with Peter's thoughts above and a couple more tweaks, that would make this:

> If you want to run an application you're developing that requires a database (or any other services), you simply define the services, their configuration, and the tasks you routinely run. Once your project is configured, you run any of the `tasks`. DevLab will spin up a parent container and link in the service containers you need.